src/lib/components/ProjectsPage/ProjectsPage.consts.ts

```diff
@@ -31,6 +31,7 @@ export const projectsTableFilters: IFilterItem<EProjectFields>[] = [
     options: Object.values(EProjectStatus).map((value) => ({
       value,
       text: PROJECT_DISPLAY_TEXTS.he.projectStatus[value],
+      isMultiOptional: true,
     })),
     defaultValue: [EProjectStatus.Active],
```

Comment:
why `isMultiOptional` is passed to every button as a prop ? why not making it general prop related to filter? you probably need to change something in the logic...